src/components/TradeWidget/index.tsx

```diff
@@ -247,39 +247,33 @@ const OrdersPanel = styled.div`
     color: #218dff;
     text-decoration: underline;
   }
+`
 
-  > button {
-    width: 1.6rem;
-    height: 100%;
-    border-right: 0.1rem solid rgba(159, 180, 201, 0.5);
-    background: #ecf2f7;
-    padding: 0;
-    margin: 0;
-    outline: 0;
+const OrdersToggler = styled.button<{ $isOpen?: boolean }>`
+  width: 1.6rem;
+  height: 100%;
+  border-right: 0.1rem solid rgba(159, 180, 201, 0.5);
+  background: #ecf2f7;
+  padding: 0;
+  margin: 0;
+  outline: 0;
 
-    @media ${MEDIA.mobile} {
-      display: none;
-    }
-
-    &::before {
-      display: block;
-      content: ' ';
-      background: url(${arrow}) no-repeat center/contain;
-      transform: rotate(180deg);
-      height: 1.2rem;
-      width: 1.6rem;
-      margin: 0;
-    }
+  @media ${MEDIA.mobile} {
+    display: none;
+  }
 
-    &:hover {
-      background-color: #ecf2f7;
-    }
+  &::before {
+    display: block;
+    content: ' ';
+    background: url(${arrow}) no-repeat center/contain;
+    height: 1.2rem;
+    width: 1.6rem;
+    margin: 0;
+    transform: ${({ $isOpen }): string => ($isOpen ? 'rotateY(180deg)' : 'rotateY(0)')};
```

Comment:
ubermeganit: only the the degrees change, so just add the actual number

Reply:
Don't need turn I don't think

Reply:
nvm u right u right